README.md

```diff
@@ -18,6 +18,10 @@ Make sure that there is a search template at
 `kagiso_search/search_results.html` in your templates folder
 (see a sample below).
 
+Add the following required configuration settings:
```

Comment:
ok then it is just the plural

On Tue, Feb 16, 2016 at 1:51 PM, colinfrankb <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Kagiso-Future-Media/kagiso_search/pull/1#discussion_r53000767>
> :
>
> > @@ -18,6 +18,10 @@ Make sure that there is a search template at
> >  `kagiso_search/search_results.html` in your templates folder
> >  (see a sample below).
> >
> > +Add the following required configuration settings:
>
> according to your code, its required, and doesn't have a default.
>
>
> https://github.com/Kagiso-Future-Media/kagiso_search/blob/master/kagiso_search/views.py#L33
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kagiso-Future-Media/kagiso_search/pull/1/files#r53000767>
> .
>